### PR TITLE
feat: bring up admin panel from local env

### PR DESCRIPTION
**Pull request by teomrd**
Enabling Netlify CMS on local env according to their [docs](https://www.netlifycms.org/docs/beta-features/#working-with-a-local-git-repository). 
The only think needed is to statically serve the admin folder.
@stathismor if you are aware of a way to serve it statically with Next, lemme know so we can avoid introducing another dependency (`serve`) just for that 

_Have to note that it didn't work with `npx netlify-cms-proxy-server`_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,15 @@ This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-opti
 
 * Preferred package manager: `npm`
 
+### Admin Panel
+
+The Admin Panel, powered by Netlify CMS, where new and existing content can be edited,
+can worked locally with the corresponding npm task
```
Comment (stathismor) [inline README.md]: nit:
```suggestion
can work locally with the corresponding npm task.
```

**Review by stathismor — COMMENTED** [review]
Have you tried `netlify-cms-proxy-server`?

https://www.netlifycms.org/docs/beta-features/#configure-the-netlify-cms-proxy-server-port-number
https://www.npmjs.com/package/netlify-cms-proxy-server

I don't mind this approach though, whatever works :smile:

**Comment by teomrd** [comment]
> Have you tried `netlify-cms-proxy-server`?
> 
> https://www.netlifycms.org/docs/beta-features/#configure-the-netlify-cms-proxy-server-port-number https://www.npmjs.com/package/netlify-cms-proxy-server
> 
> I don't mind this approach though, whatever works 😄

I did! But it's a BETA feature and possibly that's the reason it didn't work. We can proceed with `serve` for now and we can get that replaced with `netlify-cms-proxy-server` once we manage to have that working.

**Comment by stathismor** [comment]
Oh, I missed it it your comment in the PR, sorry! Looks good then!

**Review by stathismor — APPROVED** [review]

**Pull request merged**